experiments/openebs/openebs-target-container-failure/cstor_target_container_kill.yml

```diff
@@ -62,4 +62,4 @@
       - "Verified pool pods were restarted by fault injection"
       - "Before: {{ restartCount_before.stdout }}"
       - "After: {{ restartCount_after.stdout }}"
-  failed_when: "{{ restartCount_after.stdout|int }} != {{ restartCount_before.stdout|int + 1 }}"
\ No newline at end of file
+  failed_when: "{{ restartCount_after.stdout|int }} <= {{ restartCount_before.stdout|int }}"
```

Comment:
this can cause "more" restarts which were not "intended", i.e., injected by us -- and we wouldn't know because of this condition..right ?

Reply:
Yes but now we are supporting multiple container-kill after addition of interval/duration in pumba lib